webtv/README.md

```diff
@@ -0,0 +1,96 @@
+# OpenPass Sample Demo App
+
+This is a [React.js](https://create-react-app.dev/) project that creates a generic app to be compiled to multiple WebTV Technologies.
+
+This app currently implements [OpenPass Javascript SDK](https://github.com/openpass-sso/openpass-js-sdk) to do the authorization flow for the app.
+
+Current WebTV technologies supported:
+
+- [WebOS](https://www.webosose.org/docs/home/)
+- [Tizen](https://developer.tizen.org/)
+
+## Recommended Versions
+
+- Use Node.js 16
```

Comment:
Should we use a higher version? The workflow script uses 18

Reply:
It runs on node 16, but for keeping consistency just updated the docs to match the one for the workflow